src/lib/input/input.spec.ts

```diff
@@ -515,36 +516,36 @@ describe('MdInput without forms', function () {
     fixture.detectChanges();
 
     let inputEl = fixture.debugElement.query(By.css('input')).nativeElement;
-    let labelEl = fixture.debugElement.query(By.css('label')).nativeElement;
+    let formFieldEl = fixture.debugElement.query(By.css('.mat-form-field')).nativeElement;
```

Comment:
@mmalerba Taking advantage of the fact that you're working on it, would you be able to replace, `let` by `const` (in tests) when possible? It seems like almost all the variables here (in test file) aren't being reassigned and still are `let`.